data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(name, date_of_birth, escape_attempt, neutered, weight_kg) VALUES('Agumon', '2020-02-03',0, true,10.23) 
+INSERT INTO animals(name, date_of_birth, escape_attempt, neutered, weight_kg) VALUES('Gabumon', '2020-11-15',2, true,8) 
+INSERT INTO animals(name, date_of_birth, escape_attempt, neutered, weight_kg) VALUES('Pikachu', '2021-01-17',1, false, 15.04)
```

Comment:
- [ ] the date of birth for Pikachu is `2021-01-07` and not `2021-01-17` as you have entered it.